extensions/amp-story-panning-media/0.1/amp-story-panning-media.css

```diff
@@ -19,6 +19,8 @@ amp-story-panning-media {
   display: flex !important;
   justify-content: center !important;
   align-items: center !important;
+  /* For the translate3d z calculation. */
+  perspective: 1px;
```

Comment:
what happens if we don't add this? (also `!important` if we need it).

Reply:
The parent needs a perspective unit to base distance off of.
https://www.w3schools.com/cssref/css3_pr_perspective.asp

Without it, the `z` transform has no effect.